src/judoscale/core/config.py

```diff
@@ -25,12 +23,17 @@ def for_report(self):
         }
 
     def _prepare_logging(self):
+        logger = logging.getLogger('judoscale')
+        print(self.log_level)
         log_level = logging.getLevelName(self.log_level.upper())
-        logging.basicConfig(
-            level=log_level,
-            format="%(levelname)s - [Judoscale] %(message)s",
-            force=True,
-        )
+        logger.setLevel(log_level)
+
+        stdout_handler = logging.StreamHandler()
+        stdout_handler.setLevel(logging.DEBUG)
```

Comment:
I think this should be using the configured log level (`self.log_level`).

Reply:
Actually, is this line even necessary since we're setting the level for `logger`?

Reply:
No it is not, I will remove it, along with the print statement